app/models/manager_refresh/inventory_object.rb

```diff
@@ -81,45 +81,22 @@ def dependency?
       true
     end
 
-    def allowed_writers
-      return [] unless model_class
-
```

Comment:
could we still keep the automatic way, just put here

``` 
return inventory_collection.inventory_object_attributes if return inventory_collection.inventory_object_attributes
```

that should limit it to the explicitly defined set